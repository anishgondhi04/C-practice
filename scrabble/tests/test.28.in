10
5
2 2 V hello
3 1 V hello
2 2 H hello
2 6 H olleh
6 2 V olleh
3
2 o 3
2 h 3
2 e 3

