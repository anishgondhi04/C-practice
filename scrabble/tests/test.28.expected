Invalid word placement: (3,1) V, hello
..........
..........
..hello...
..e...l...
..l...l...
..l...e...
..olleh...
..........
..........
..........
Place vertically at (3,2)
Place horizontally at (2,3)
Place horizontally at (6,5)
Place vertically at (5,6)
